js/controllers/authors.js

```diff
@@ -0,0 +1,69 @@
+/* global app */
+/* global angular */
+
+app.controller('authorsCtrl', ['$scope', 'authorService', 'enableEditService', 'bookService', function($scope, authorService, enableEditService, bookService, $index) {
+  
+  $scope.include = function(obj) {
+     authorService.updateAuthors(obj);
+   };
+   
+  $scope.temp = authorService.authorsList;
+  $scope.authorErrorMessage = "";
+  
+  $scope.edit = enableEditService.edit;
+  
+  $scope.gridOptions = {
+    
+    data: 'temp',
+    enableCellSelection: true,
+    enableCellEdit: true,
+    enableCellEditOnFocus: true,
+    rowEditWaitInterval: -1,
```

Comment:
in general your code is very light on comments - you should explain why your code is structured the way it is. In particular here, why is this set to -1?

Reply:
added comments back in - rowEditWaitInterval: -1 prevents ui-grid's default save behavior